include/ringmesh/geomodel/geomodel_builder_topology.h

```diff
@@ -193,15 +202,17 @@ namespace RINGMesh
         friend class GeoModelBuilder< 2 >;
 
     public:
-        void add_mesh_entity_boundary_relation(
-            const gmme_id& incident_entity_id,
-            const gmme_id& boundary_id,
-            bool side = false ) override;
 
-        void set_mesh_entity_boundary( const gmme_id& gmme,
+        void add_surface_line_boundary_relation(
+            index_t incident_surface_id,
+            index_t boundary_line_id,
+            bool side );
+
+        void set_surface_line_boundary(
+            index_t incident_surface_id,
             index_t id,
```

Comment:
Could you improve all the "id" variables in the "set_....boundary". Maybe something like boundary_id or something even better. At reading, this id is not clear.